docs/index.md

```diff
@@ -205,44 +259,170 @@ Add/Del/Topic: https://github.com/wechaty/wechaty/issues/32
+<a name="Contact+city"></a>
+
+### contact.city() ⇒ <code>string</code> &#124; <code>undefined</code>
+Get the region 'city' from a contact
+
+**Kind**: instance method of <code>[Contact](#Contact)</code>  
 <a name="Contact+avatar"></a>
 
-### contact.avatar()
+### contact.avatar() ⇒ <code>Promise.&lt;NodeJS.ReadableStream&gt;</code>
 Get avatar picture file stream
 
 **Kind**: instance method of <code>[Contact](#Contact)</code>  
+<a name="Contact+refresh"></a>
+
+### contact.refresh() ⇒ <code>Promise.&lt;this&gt;</code>
+Force reload data for Contact
+
+**Kind**: instance method of <code>[Contact](#Contact)</code>  
+<a name="Contact+self"></a>
+
+### contact.self() ⇒ <code>boolean</code>
+Check if contact is self
+
+**Kind**: instance method of <code>[Contact](#Contact)</code>  
+**Returns**: <code>boolean</code> - True for contact is self, False for contact is others  
+<a name="Contact+say"></a>
+
+### contact.say(content) ⇒ <code>Promise.&lt;void&gt;</code>
+Say `content` to Contact
+
+**Kind**: instance method of <code>[Contact](#Contact)</code>  
+
+| Param | Type |
+| --- | --- |
+| content | <code>string</code> | 
+
 <a name="Contact.findAll"></a>
 
-### Contact.findAll()
-find contact by `name` or `alias`
+### Contact.findAll([queryArg]) ⇒ <code>Promise.&lt;Array.&lt;Contact&gt;&gt;</code>
+If use Contact.findAll() get the contact list of the bot.
 
 **Kind**: static method of <code>[Contact](#Contact)</code>  
+
+| Param | Type |
+| --- | --- |
+| [queryArg] | <code>ContactQueryFilter</code> | 
+
 <a name="Contact.find"></a>
 
-### Contact.find()
-try to find a contact by filter: {name: string | RegExp}
+### Contact.find(query) ⇒ <code>Promise.&lt;(Contact\|null)&gt;</code>
+Find contact by name or alias, if the result more than one, return the first one.
 
 **Kind**: static method of <code>[Contact](#Contact)</code>  
+**Returns**: <code>Promise.&lt;(Contact\|null)&gt;</code> - If can find the contact, return Contact, or return null  
+
+| Param | Type |
+| --- | --- |
+| query | <code>ContactQueryFilter</code> | 
+
+**Example**  
+```
+const contactFindByName = await Contact.find({ name:"ContactName"} )
+const contactFindByAlias = await Contact.find({ alias:"ContactAlias"} )
+```
+<a name="Contact.load"></a>
+
+### Contact.load(id) ⇒ <code>[Contact](#Contact)</code>
+Load data for Contact by id
+
+**Kind**: static method of <code>[Contact](#Contact)</code>  
+
+| Param | Type |
+| --- | --- |
+| id | <code>string</code> | 
+
+<a name="Gender[undefined]"></a>
+
+## Gender[undefined]
+The Unknown value
+
+**Kind**: global variable  
+<a name="Gender[undefined]"></a>
+
+## Gender[undefined]
+The Male value
+
+**Kind**: global variable  
+<a name="Gender[undefined]"></a>
+
+## Gender[undefined]
+The Female value
+
+**Kind**: global variable  
+<a name="Gender"></a>
+
+## Gender : <code>enum</code>
+Enum for Gender values.
+
+**Kind**: global enum  
+**Read only**: true  
+**Export**:   
 <a name="config_1"></a>
 
 ## config_1
```

Comment:
After I modified as you requested, it has changed automatically, maybe it is correct now.